taskcluster/ci/build/kind.yml

```diff
@@ -0,0 +1,108 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - fenix_taskgraph.transforms.build:transforms
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+job-defaults:
+    description: Build Fenix from source code.
+    treeherder:
+        kind: build
+        symbol: B
+        platform: android-all/opt
+        tier: 1
+    worker-type: b-android
+    worker:
+        docker-image: {in-tree: base}
+        max-run-time: 7200
+        chain-of-trust: true
+    run:
+        using: gradlew
+        use-caches: false
+    run-on-tasks-for: []
+    # Builds generate multiple APKs with different ABIs. For each APK described
+    # by `gradlew printVariant`, an artifact will be generated. `variant` and
+    # the per-apk config from `printVariant` can be used as subsistutions in
```

Comment:
substitutions* :)
Thanks for documenting which properties have variables available :)

Reply:
Thanks! I intend to change the gradle call because it slowed down the decision task run. The reason is: taskgraph generates *all* tasks (before filtering them out), so we have about 8 call to gradle, which is slower than before. I think I'll replace it with a single call to gradle which outputs config.